From 1df9b73db6541cbc483c6ee1b21dbe0cb6e5cbfa Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Wed, 29 Jul 2020 08:20:41 +0200
Subject: [PATCH] meson: do not check for c++

Wayland does not need c++, fixes building with toolchains lacking c++.

Backported from upstream PR:
https://gitlab.freedesktop.org/wayland/wayland/-/merge_requests/88

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index b67b101..e47a52d 100644
--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project(
-	'wayland', 'c', 'cpp',
+	'wayland', 'c',
 	version: '1.18.0',
 	license: 'MIT',
 	meson_version: '>= 0.47.0',
-- 
2.27.0

